ci-operator/config/openshift-metal3/dev-scripts/openshift-metal3-dev-scripts-master__packetcheck.yaml

```diff
@@ -7,7 +7,7 @@ resources:
   '*':
     requests:
       cpu: 100m
-      memory: 200Mi
+      memory: 201Mi
```

Comment:
As an alternative workaround the workflow, did you try to comment the `cron` directive?

Reply:
That worked, thank you.